Doc/library/socket.rst

```diff
@@ -289,6 +289,16 @@ Constants
 
    .. versionadded:: 3.2
 
+.. data:: SOCK_TYPE_MASK
+
+   On Linux, ``socket.type`` int field combines the type of the socket
```

Comment:
Please describe what `SOCK_TYPE_MASK` is and hint on proper its usage.

Reply:
I'm not even sure we should expose it, since `socket.type` takes care of it with this PR.

Reply:
To clarify: `socket.type` now applies it, so there's no real use case for `SOCK_TYPE_MASK` anymore.

Reply:
Yes, I see.
Ok wish not exposing the mask, but if it is exposed -- doc should be updated.
Honestly I don't see how the mask can be useful -- at Python level socket type is always masked already.